src/NuGet.Clients/PackageManagement.VisualStudio/IDE/IVsSolutionManager.cs

```diff
@@ -46,5 +46,11 @@ public interface IVsSolutionManager : ISolutionManager
         /// </summary>
         /// <returns>Deferred prokects file path.</returns>
         Task<IEnumerable<string>> GetDeferredProjectsFilePathAsync();
+
+        /// <summary>
+        /// Returns number of nominations still pending for .net core projects.
+        /// </summary>
+        /// <returns>Number of pending nominations.</returns>
+        int GetExpectedProjectsNomination();
```

Comment:
hmm makes sense! earlier i was validating changes with previous run so used int.. but now it makes more sense to just use boolean.